.github/CODEOWNERS

```diff
@@ -1 +1,2 @@
+* @viamrobotics/sdk-netcode
 * @stuqdog
```

Comment:
do you still want to keep yourself as a code-owner?

Reply:
good question! probably not necessary, I'll update :)

Reply:
[opt] separately I was curious why you didn't automatically get added a code reviewer while you were an individual code owner - does the action randomize from the full pool of code-owners?

Reply:
I don't think I can be added as a reviewer to my own PR?